guides/rest_api.md

```diff
@@ -656,3 +657,49 @@ You should delete a task after it is done. Eventually done tasks will clear out
 ```
 DELETE /api/rest/v1/tasks/:id
 ```
+
+## Convert Chapters
+
+
+```
+POST /api/rest/v1/convert/chapters
+```
+
+The chapters converter endpoint takes a string of one of the following formats:
+
+- PSC (Podlove Simple Chapters)
+- mp4chaps
+- JSON
+
+The input format is detected automatically.
+
+**Parameters**
+
+| Name            | Type     | Description                          |
+| --------------- | -------- | ------------------------------------ |
+| `data`          | `string` | **Required.** raw chapters content   |
+| `output_format` | `string` | json, psc or mp4chaps. default: json |
+
+
+The response content type depends on the chosen `output_format`. In case of errors, the response is JSON.
+
+**json**
+
+Content Type: `application/json; charset=utf-8`
+
+The output is a list of chapters with the following keys:
+
+| Name    | Type      | Description                        |
+| ------- | --------- | ---------------------------------- |
+| `image` | `string`  | chapter image URL. null if empty   |
+| `link`  | `string`  | null if empty                      |
+| `start` | `integer` | chapter start time in milliseconds |
+| `title` | `string`  | chapter title                      |
```

Comment:
Hmm. In my head this was different:
a) there was code point that would return chapters in one of the formats e.g. `GET /api/rest/v1/audios/:audio_id/chapters` would grow an optional format argument that would return json, psc, mp4chaps or the database format from the db
b) the convert code path would only ever return our database format sans the audio_id. Which incidentally this is, sans `start_string` and `duration`

Also I do think you probably should put in the `duration` and `duration_string` to the output as well if you are in the vicinity.


P.S.: 
There is also a docu bug in the rest_api file with chapters update (the last entry of chapter on the update section should be `chapter[image]` type `file`.

Reply:
Referencing #175 here for easy closing.